flakytests/job-submission/18-check-chunking.t

```diff
@@ -15,36 +15,44 @@
 # You should have received a copy of the GNU General Public License
 # along with this program.  If not, see <http://www.gnu.org/licenses/>.
 #-------------------------------------------------------------------------------
-
 # Test that a job containing more than 100 tasks will split into batches.
 
 . "$(dirname "$0")/test_header"
+set_test_number 3
 
-set_test_number 2
+create_test_globalrc '' '
+process pool size = 1
+'
```

Comment:
The hundreds of simultaneous jobs were killing my own desktop. I can see why Travis CI was so unhappy. (Fork bomb. Memory issue.) I am now restricting the subprocess pool to run 1 command at a time for this test.